en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,90 @@
+Customize Content Attribute Display
+===================================
+
+Content attributes type are mixed, it's difficult to render them to user. When we
+want to display a string in datatable or in other place, value give array or object.
+To avoid the problem, Open Orchestra add a variable string in HTML Format to
+content attributes. Before to save the attribute, Open Orchestra use a
+``ValueTransformerManager`` to convert value into string.
```

Comment:
it's more a generation rather than a conversion as the value is still available after the operation.